### PR TITLE
test: add e2e test cases of user, role, and plugin

### PR DESCRIPTION
**Pull request by LinuxSuRen**
#### What type of PR is this?

/kind test

#### What this PR does / why we need it:
As mentioned in the title.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: e2e/testsuite.yaml [hunk 1]
```diff
@@ -3,6 +3,9 @@ api: |
   {{default "http://halo:8090" (env "SERVER")}}/apis
 param:
   postName: "{{randAlpha 6}}"
+  userName: "{{randAlpha 6}}"
+  roleName: "{{randAlpha 6}}"
+  auth: "Basic YWRtaW46MTIzNDU2"
```
[the remark this answers is not in the document]
Comment (LinuxSuRen) [reply e2e/testsuite.yaml:8]: The user in the parameter `auth` is the inital user. But the `username` was created by API. So, I think they should be different.

[the remark this answers is not in the document]
Comment (guqing) [reply e2e/testsuite.yaml:8]: Oh, please ignore it. I just ignored the case of createUser

**Review by ruibaby — COMMENTED** [review]
/lgtm

图片

**Review by guqing — APPROVED** [review]

**Comment by ruibaby** [comment]
It seems that because the workflow file was modified, Prow is unable to automatically merge, so I will manually merge this PR.

**Pull request merged**